network.go

```diff
@@ -449,6 +461,16 @@ func (n *network) UnmarshalJSON(b []byte) (err error) {
 	}
 	n.name = netMap["name"].(string)
 	n.id = netMap["id"].(string)
+	if v, ok := netMap["created"]; ok {
+		// n.created is time.Time but marshalled as string
+		s, ok := v.(string)
+		if !ok {
```

Comment:
Can't we skip the type check given we know it can only be a string, since it is us that write it to store ?
In the past we did not have any "created" field being saved to store.